### PR TITLE
Release 6.0.0

### PR DESCRIPTION
**Pull request by crow**
Release 6.0.0

Manually tested android and ios:
android_manual_test
ios_manual_test

### CODE REVIEW

**Comment by crow** [comment]
this won't pass until the requisite sdks are out

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,17 @@
 
+Version 6.0.0 - December 19, 2019
+=================================
+Major release to update iOS to modularized SDK 13.0.2, update Android SDK to 12.1.0,
```
Comment (rlepinski) [inline CHANGELOG.md]: 13.0.3

**Comment by crow** [comment]
will merge this momentarily

File: android/src/main/java/com/urbanairship/reactnative/UrbanAirshipReactModule.java [hunk 2]
```diff
@@ -126,6 +127,8 @@ public void onHostDestroy() {
         });
 
         EventEmitter.shared().attachReactContext(getReactApplicationContext());
+
+        UAirship.shared().getAnalytics().registerSDKExtension(Analytics.EXTENSION_REACT_NATIVE, MODULE_VERSION);
```
Comment (rlepinski) [inline android/src/main/java/com/urbanairship/reactnative/UrbanAirshipReactModule.java]: Minor nit, but this should probably be set in Autopilot airshipReady method

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,17 @@
 
+Version 6.0.0 - December 30, 2019
+=================================
+Major release to update iOS to modularized SDK 13.0.4, update Android SDK to 12.1.0,
+add cross-platform attribute support and modernize the Message Center Javascript
+and Typescript interfaces.
+
+Changes
+-------
+- Updated iOS SDK to 13.0.3
```
Comment (oristanovic) [inline CHANGELOG.md]: Shouldn't it be 13.0.4?

File: ios/UARCTModule/UARCTAutopilot.m [hunk 4]
```diff
@@ -1,12 +1,13 @@
...
 
 #import "UARCTAutopilot.h"
 #import "UARCTEventEmitter.h"
 #import "UARCTDeepLinkAction.h"
 #import "UARCTMessageCenter.h"
+#import "UARCTModuleVersion.h"
 
 NSString *const UARCTPresentationOptionsStorageKey = @"com.urbanairship.presentation_options";
-NSString *const UARCTAirshipRecommendedVersion = @"13.0.1";
+NSString *const UARCTAirshipRecommendedVersion = @"13.0.3";
```
Comment (oristanovic) [inline ios/UARCTModule/UARCTAutopilot.m]: Shouldn't it be 13.0.4?

**Pull request merged**